.github/workflows/pr-image-tests.yaml

```diff
@@ -79,3 +80,33 @@ jobs:
           # Ensure we disable buildkit
           DOCKER_BUILDKIT: 0
         shell: bash
+    pr-image-tests-build-windows-images:
+      name: PR - Docker windows build test, windows 2019
+      runs-on: windows-2019
+      permissions:
+        contents: read
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v3
+
+        - name: Extract metadata from Github
+          id: meta
+          uses: docker/metadata-action@v4
+          with:
+            images: ${{ github.repository }}/pr-${{ github.event.pull_request.number }}
+            tags: |
+              type=sha
+            flavor: |
+              suffix=-windows
+    
+        - name: Build the windows images
+          id: build
+          run: |
+            docker build -t ${{ steps.meta.outputs.tags }} -f ./dockerfiles/Dockerfile.windows .
```

Comment:
Let's add in the build-arg here @celalettin1286 to make it explicit this must be built both on and for Windows 2019